### ISSUE TITLE
I just started playing agien and this Happend

### ISSUE THREAD

**Issue by NetflixAnChill** [Wynncraft/Issues#5119]
I haven't played for about 2-3 years now and came into the reset room and dident know about how you have to use your stats to wear armor now I cant wear any thing that can protect me. My account is a level 52 archer and I can only use a level 17 bow I was wondering if there is any way to get a reset for my skill points and be able to use my items, My account name is Miner4life79 and this is the only character that has this problem

**Comment by Moe-Ronickah** [comment]
After not playing for 2-3 years, I'm very curious as to why you didn't read the forums, first, to see what changes had been made in that time.  However, there is a quest, Tower of Amnesia, that you can do in order to get a skill reset scroll.  By the time that you complete that quest, you will have obtained plenty of new armor and weapons, anyway.  The game has dramatically changed since you last played; quests, new provinces, the level cap went from 70 to 75 and now to 100, just to start.

In short, no, there is no way for anyone to give you a reset scroll, unless you find someone in game to give you one, and even then, you won't really know what you are doing, so you have two options; 1) keep playing with your lv52 archer, or 2) start a new class and learn the new mechanics of the game.
